engine/runtime/src/main/scala/org/enso/interpreter/instrument/execution/JobExecutionEngine.scala

```diff
@@ -36,10 +35,7 @@ final class JobExecutionEngine(
 
   private val context = interpreterContext.executionService.getContext
 
-  private val jobParallelism =
-    interpreterContext.executionService.getContext.getEnvironment.getOptions
-      .get(RuntimeServerInfo.JOB_PARALLELISM_KEY)
-      .intValue()
+  private val jobParallelism = context.getJobParallelism
```

Comment:
`JOB_PARALLELISM_KEY` is registered in `RuntimeServerInstrument`. This option descriptor key should be contained in the context. Unless there is something wrong with the initialization of `RuntimeServerInstrument`?